shared/js/background/tracker-lists.es6.js

```diff
@@ -14,15 +13,19 @@ function setList (name, data) {
 function loadLists(){
     var listLocation = constants.trackerListLoc
     var blockLists = constants.blockLists
+    console.log("LOAD LISTS")
+
     blockLists.forEach( function(listName) {
+        console.log(`Load: ${listName}`)
+
         load.JSONfromLocalFile(listLocation + "/" + listName, (listJSON) => {
             console.log(`Loaded tracker list: ${listLocation}/${listName}`)
             lists[listName.replace('.json', '')] = listJSON
         })
     })
 }
 
-settings.ready().then(() => loadLists())
+loadLists()
 
 module.exports = {
```

Comment:
This whole thing simply loads a local JSON file. Let's just replace it with a `require()`.

Reply:
Yeah good idea. That would make my life easier too.